install_test.sh

```diff
@@ -1,17 +1,42 @@
-#!/bin/sh
+#!/bin/bash
 
 set -ev
 
-# Lint.
-shellcheck -s sh ./*.sh
-shfmt -d .
+# Install shfmt.
+shfmt_version=v2.6.2
+case $(uname -s) in
+Darwin) shfmt_os="darwin" ;;
+*) shfmt_os="linux" ;;
+esac
+shfmt_url="https://github.com/mvdan/sh/releases/download/${shfmt_version}/shfmt_${shfmt_version}_${shfmt_os}_amd64"
+curl -sSL -o ./shfmt "$shfmt_url"
```

Comment:
I don't want to download shfmt when I run it locally. Can you leave that in the travis script?

Reply:
I didn't run this test locally. For a local test I always just ran `./install.sh && ~/.deno/bin/deno -v`.

But sure, I can move it back if you want.

Reply:
In the .yaml file, there'd be duplication since there are seperate sections for osx and linux; in the script, it's just a case statement.

Would it be okay if I put the shfmt download in a "if CI" block?

Reply:
Ok, sure - but use this:

```sh
if ! command -v shfmt; then
  echo "shfmt not installed"
fi
```

Reply:
Actually, I removed it from the script now and put it into the .travis.yml file since the installation locations on linux and osx are different.